plugins/source/stripe/resources/services/accounts/accounts.go

```diff
@@ -0,0 +1,41 @@
+// Code generated by codegen; DO NOT EDIT.
+
```

Comment:
Might be controversial, but with how little code is in here now, and how few tables there are, I don't think we're winning much by making it generated (unless we can generate 50+ more). Could we remove the codegen for this plugin entirely, to make it easier to understand for outside contributors?

Reply:
Agree

Reply:
We can generate 50+ more because go-stripe is pretty consistent (except for `stripe.InvoiceItem` being `invoiceitem`, but that's something to do with the API itself) and with Stripe since we don't have a base installation to actually test everything going the codegen way seems superior. Any outside contributor can add 2 lines to a recipe file and run `make gen` to automatically implement any root level resource including mock fixtures. We don't need to treat everything as if they are nails, even if we have a hammer at our disposal.

Reply:
If it's all easily generatable why not generate all the tables right now? I think the best practice should be generate everything if it's easily generatable, if not then just do a few important ones and don't use codegen.

Reply:
Using half codegen looks like an interesting compromise where we generate the resolvers and other table defs, but don't make it more complicated by adding all the codegen.Table* logic.